pkgs/misc/g810-led/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, hidapi
+, profile ? "/etc/g810-led/profile"
+}:
+
+stdenv.mkDerivation rec {
+  pname = "g810-led";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "MatMoul";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ymkp7i7nc1ig2r19wz0pcxfnpawkjkgq7vrz6801xz428cqwmhl";
+  };
+
+  buildInputs = [ hidapi ];
+
+  installPhase = ''
+    install -D bin/g810-led $out/bin/g810-led
+    ln -s \./g810-led $out/bin/g213-led
+    ln -s \./g810-led $out/bin/g410-led
+    ln -s \./g810-led $out/bin/g413-led
+    ln -s \./g810-led $out/bin/g512-led
+    ln -s \./g810-led $out/bin/g513-led
+    ln -s \./g810-led $out/bin/g610-led
+    ln -s \./g810-led $out/bin/g815-led
+    ln -s \./g810-led $out/bin/g910-led
```

Comment:
Why `ln -s` and not `cp`? Or `install`?

Reply:
So we don't waste space having the same binary multiple times, this is also how it's done in the original makefile: https://github.com/MatMoul/g810-led/blob/master/makefile

Reply:
It would have been ideal if you'd patch upstream's makefile and make it use `PREFIX` properly and then use their makefile as is. The lines here are wrong:

https://github.com/MatMoul/g810-led/blob/5ee810a520f809e65048de8a8ce24bac0ce34490/makefile#L55-L74

They should use `$(PREFIX)` there instead of `$(DESTDIR)/usr` - and it's the default value of `$(PREFIX)` according to:

https://github.com/MatMoul/g810-led/blob/5ee810a520f809e65048de8a8ce24bac0ce34490/makefile#L13

Could you contact upstream about it?